src/main/java/io/dropwizard/revolver/resource/RevolverRequestResource.java

```diff
@@ -440,11 +440,6 @@ private Response executeCallbackSync(final String service, final RevolverHttpApi
         );
         persistenceProvider.setRequestState(requestId, RevolverRequestState.REQUESTED, mailBoxTtl);
         val result = response.get();
-        callbackHandler.handle(requestId, RevolverCallbackResponse.builder()
```

Comment:
@phaneesh Added this, thinking that callback is not going in SYNC mode. But we are sending callback inside /v1/callback api
This resulted into multiple callbacks for nexus. So removed, this